lib/site_prism/element_container.rb

```diff
@@ -147,7 +147,7 @@ def create_nonexistence_waiter(element_name, *find_args)
       method_name = "wait_for_no_#{element_name}"
       create_helper_method(method_name, *find_args) do
         define_method(method_name) do |timeout = nil, *runtime_args|
-          timeout = timeout.nil? ? Waiter.default_wait_time : timeout
+          timeout = timeout.nil? ? Capybara.default_max_wait_time : timeout
```

Comment:
How about writing like this?

```
define_method(method_name) do |timeout = Capybara.default_max_wait_time, *runtime_args|
```

The `create_visibility_waiter` method just below it is written as such.

Reply:
Indeed it is, not sure why. Will standardise up